### ISSUE TITLE
bootstrap_stats method runs parametric bootstrap

### ISSUE THREAD

**Issue by U-R-dev** [Labo-Lacourse/stepmix#57]
First of all, thank you for providing such a wonderful package. As stepmix is the only open source program that can perform LCA with distal outcomes, I've been using it for months now.

After upgrading from version 2.1.3 to 2.2.0, we found that nonparametric bootstrapping using bootstrap_stats took an enormous amount of time, about 10 times longer than before. Step-by-step debugging revealed that the variable "parametric" was set to "True" and, in fact, parametric bootstrapping was performed.

I could reproduce the above with the following simple sample code on the Google Colab.

```python
!pip install stepmix ipdb
from stepmix.datasets import data_bakk_response
from stepmix.stepmix import StepMix

# Simulate data
X, Y, _ = data_bakk_response(n_samples=2000, sep_level=.9, random_state=42)

# Define base model
model = StepMix(n_components=3, n_steps=1, measurement='bernoulli',
                structural='gaussian_unit', random_state=42)
model.fit(X, Y)

# Bootstrap
breakpoint()  # for debugging
stats_dict = model.bootstrap_stats(X, Y, n_repetitions=5)
```

As shown in the debugging screenshot below,

<img width="664" alt="Screenshot 2024-02-15 at 15 01 33 2" src="https://github.com/Labo-Lacourse/stepmix/assets/58283199/cba00c73-55b7-4e1b-b0fb-801830890373">

when the command reached the bootstrap function (line 1084), the variable "parametric" was set to "True" and remained "True" until the end of the process, resulting in conducting parametric bootstrapping.

I don't understand why default value, parametric=False, is ignored, but It would be helpful if you could look into this phenomenon.

Thank you,

**Comment by sachaMorin** [comment]
Thanks for reporting. I'll look into this.

**Comment by sachaMorin** [comment]
The `progress_bar` argument  of `self.bootstrap_stats` is wrongly passed to `self.bootstrap` as the `parametric` argument because of positional arguments and the changes introduce in 2.2.0. Good catch! I'll release a patch today.

**Comment by sachaMorin** [comment]
Should be fixed as of 2.2.1. Also if by any chance you are in a position to share your work, please do! We are always curious to see how our users are using StepMix.

**Comment by U-R-dev** [comment]
Ah, I see. It's so basic, but I didn't notice it. Thank you for the quick response. We will be sure to cite your paper when our work is published!
